src/components/tables/DataTable.vue

```diff
@@ -527,18 +528,15 @@ watch(search, () => {
                     {{ column[columnAttr] }}
                   </span>
 
-                  <template v-if="column.align === 'right'" #prepend>
+                  <template v-if="column.align === 'end'" #prepend>
                     <Icon
                       :class="css.sort__icon"
                       name="arrow-down-line"
                       size="18"
                     />
                   </template>
 
-                  <template
-                    v-if="!column.align || column.align === 'left'"
-                    #append
-                  >
+                  <template v-if="column.align !== 'end'" #append>
```

Comment:
let's simplify the entire `template` tag here:

```html
<template #append>
  <Icon
    v-if="column.align != 'end'"
    :class="css.sort__icon"
    name="arrow-down-line"
    size="18"
  />
  <Chip
    v-if="getSortIndex(column.key) >= 0"
    :label="`${getSortIndex(column.key) + 1}`"
    size="sm"
    color="grey"
  />
</template>
```

Reply:
nice catch!! updated